>From 65f237e8f935055b82455c408587687cfc6633f8 Mon Sep 17 00:00:00 2001
From: Michael Stapelberg <stapelberg@debian.org>
Date: Wed, 14 Aug 2013 14:10:18 +0200
Subject: [PATCH] allow Vcs-Upstream-* tags

Just as the existing Vcs-* tags, these point to a version control
repository (and corresponding browser), but for the upstream repository
instead of the Debian packaging repository.
---
 scripts/Dpkg/Control/FieldsCore.pm |   27 +++++++++++++++++++++++++++
 1 file changed, 27 insertions(+)

diff --git a/scripts/Dpkg/Control/FieldsCore.pm b/scripts/Dpkg/Control/FieldsCore.pm
index 44d90d1..ef21dc1 100644
--- a/scripts/Dpkg/Control/FieldsCore.pm
+++ b/scripts/Dpkg/Control/FieldsCore.pm
@@ -315,6 +315,33 @@ our %FIELDS = (
     'Vcs-Svn' => {
         allowed => ALL_SRC,
     },
+    'Vcs-Upstream-Browser' => {
+        allowed => ALL_SRC,
+    },
+    'Vcs-Upstream-Arch' => {
+        allowed => ALL_SRC,
+    },
+    'Vcs-Upstream-Bzr' => {
+        allowed => ALL_SRC,
+    },
+    'Vcs-Upstream-Cvs' => {
+        allowed => ALL_SRC,
+    },
+    'Vcs-Upstream-Darcs' => {
+        allowed => ALL_SRC,
+    },
+    'Vcs-Upstream-Git' => {
+        allowed => ALL_SRC,
+    },
+    'Vcs-Upstream-Hg' => {
+        allowed => ALL_SRC,
+    },
+    'Vcs-Upstream-Mtn' => {
+        allowed => ALL_SRC,
+    },
+    'Vcs-Upstream-Svn' => {
+        allowed => ALL_SRC,
+    },
     'Vendor' => {
         allowed => CTRL_FILE_VENDOR,
     },
-- 
1.7.10.4

